core/ecschema-metadata/src/Deserialization/JsonProps.ts

```diff
@@ -34,6 +34,17 @@ export interface SchemaProps {
   readonly customAttributes?: Array<{ [value: string]: any }>;
 }
 
+/**
+ * JSON Object interface used to deserialize into a [[SchemaKey]].
+ * @beta
+ */
+export interface SchemaKeyProps {
+  readonly name: string;

```

Comment:
Document these and re-run extract-api. It is not obvious that they are version numbers.